### PR TITLE
[INLONG-10826][SDK] Transform support TRIM(), REPLICATE() function

### LINKED ISSUE #10826: [Feature][SDK] Transform support TRIM(), REPLICATE() function
### Description

* Parent issue https://github.com/apache/inlong/issues/10796
TRIM(s)：
Remove Spaces before and after the string.
REPLICATE(s, count)
Repeat a string multiple times and return a new string

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by emptyOVO**
Fixes #10826 

### Motivation

Add two string function class：ReplicateFunction, TrimFunction.
Add the corresponding unit test codes

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [x] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### CODE REVIEW

File: inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplicateFunction.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        Object stringObj = stringParser.parse(sourceData, rowIndex, context);
+        Object countObj = countParser.parse(sourceData, rowIndex, context);
+        String str = OperatorTools.parseString(stringObj);
+        double count = OperatorTools.parseBigDecimal(countObj).doubleValue();
+        return repeat(str, count);
+    }
+    private String repeat(String str, double count) {
+        String repeatString = "";
+        if (count == 0)
+            return "";
```
Comment (aloyszhang) [inline inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplicateFunction.java:62]: add  braces

File: inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplicateFunction.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        if (count == 1)
+            return str;
+        while (count > 0) {
+            if (count % 2 != 0) {
+                repeatString += str;
```
Comment (aloyszhang) [inline inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplicateFunction.java:67]: StringBuilder is better

File: inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplicateFunction.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+import org.apache.inlong.sdk.transform.process.Context;
+import org.apache.inlong.sdk.transform.process.operator.OperatorTools;
+import org.apache.inlong.sdk.transform.process.parser.ValueParser;
+
+import net.sf.jsqlparser.expression.Expression;
+import net.sf.jsqlparser.expression.Function;
+
+import java.util.List;
+
+public class ReplicateFunction implements ValueParser {
```
Comment (aloyszhang) [inline inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplicateFunction.java:30]: add descriptions like other functions did

File: inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplicateFunction.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        List<Expression> expressions = expr.getParameters().getExpressions();
+        stringParser = OperatorTools.buildParser(expressions.get(0));
+        countParser = OperatorTools.buildParser(expressions.get(1));
+    }
+    /**
+     * parse
+     * @param sourceData
+     * @param rowIndex
+     * @return
+     */
```
[the remark this answers is not in the document]
Comment (emptyOVO) [reply inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplicateFunction.java]: done

File: inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplicateFunction.java (hunk 1 above)
Comment (emptyOVO) [reply inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplicateFunction.java:55]: done

File: inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplicateFunction.java (hunk 2 above)
Comment (emptyOVO) [reply inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplicateFunction.java]: ok, i will fix it

Comment (emptyOVO) [reply inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplicateFunction.java]: done

File: inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplicateFunction.java (hunk 3 above)
Comment (emptyOVO) [reply inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplicateFunction.java:33]: done

**Review by aloyszhang — APPROVED** [review]

**Review by vernedeng — APPROVED** [review]

**Pull request merged**